coap-gateway/coapconv/grpcCode2CoapCode.go

```diff
@@ -5,17 +5,24 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
-func GrpcCode2CoapCode(statusCode codes.Code, method coapCodes.Code) coapCodes.Code {
+type Operation uint8
+
+const Create_Operation Operation = 1
```

Comment:
Why not just Create,Retrieve,Update,Delete? And where is Observe?

Reply:
Observe is not used